### DISCUSSION TITLE
求助!!在汉化的时候在替换stain这个文件的时候会有程式错误

### DISCUSSION THREAD

**Discussion by BangCockk** [GpointChen/FFXIVChnTextPatch-GP#336] [General]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134466621/8c31dd86-7e0e-42b5-98f5-f710e89b0be7)
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134466621/5fc8f586-1681-4fe2-821f-4cf9f27c1180)
然后很多基本的文本(不是最近更新的)还是英文

**Comment by GpointChen** [comment]
這個問題應該在最新版中已被修復，請用最新版。

**Reply by BangCockk** [reply]
github好像不能直接下载文件夹

**Reply by GpointChen** [reply]
什麼文件夾？

**Reply by BangCockk** [reply]
是要下载那个resource文件夹吗

**Reply by GpointChen** [reply]
請到[Release頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases)下載最新版

**Reply by BangCockk** [reply]
> 請到[Release頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases)下載最新版

非常感谢！大佬加油🥰
